package.json

```diff
@@ -12,10 +12,11 @@
     "build:preview": "vite preview",
     "build:readme": "node tools/update-readme.js",
     "start": "vite",
+    "build": "tsc",
```

Comment:
Oops, it looks like we already have a `build` script above. Let's rename to `build:types` to disambiguate.